user/apache-arrow/template.py

```diff
@@ -0,0 +1,130 @@
+pkgname = "apache-arrow"
+pkgver = "20.0.0"
+pkgrel = 0
+# cpp/thirdparty/versions.txt
+_mimalloc_ver = "2.0.6"
+_arrowsha = "d2a13712303498963395318a4eb42872e66aead7"
+_parquetsha = "18d17540097fca7c40be3d42c167e6bfad90763c"
+build_wrksrc = "cpp"
+build_style = "cmake"
+configure_args = [
+    "-DARROW_BUILD_EXAMPLES=OFF",
+    "-DARROW_BUILD_STATIC=OFF",
+    "-DARROW_BUILD_TESTS=ON",
+    "-DARROW_DEPENDENCY_SOURCE=SYSTEM",
+    "-DARROW_COMPUTE=ON",
+    "-DARROW_CSV=ON",
+    "-DARROW_DATASET=ON",
+    "-DARROW_FILESYSTEM=ON",
+    "-DARROW_FLIGHT=ON",
+    "-DARROW_HDFS=ON",
+    "-DARROW_JSON=ON",
+    "-DARROW_JSON=ON",
+    "-DARROW_ORC=OFF",
+    "-DARROW_PARQUET=ON",
+    "-DARROW_SIMD_LEVEL=NONE",
+    "-DARROW_TENSORFLOW=ON",
+    "-DARROW_USE_GLOG=ON",
+    "-DARROW_USE_LLD=ON",
+    "-DARROW_WITH_BROTLI=ON",
+    "-DARROW_WITH_BZ2=ON",
+    "-DARROW_WITH_LZ4=ON",
+    "-DARROW_WITH_MUSL=ON",
+    "-DARROW_WITH_SNAPPY=ON",
+    "-DARROW_WITH_ZLIB=ON",
+    "-DARROW_WITH_ZSTD=ON",
+    "-DPARQUET_REQUIRE_ENCRYPTION=ON",
+    "-DLZ4_ROOT=/usr",
+]
+make_check_args = [
+    "-E",
+    "arrow-array-test|arrow-compute-scalar-temporal-test|arrow-utility-test|arrow-csv-test|arrow-ipc-json-simple-test|arrow-dataset-file-parquet-test",
+]
+make_check_env = {
+    "ARROW_TEST_DATA": f"/builddir/apache-arrow-{pkgver}/testing/data",
```

Comment:
there is no guarantee these kind of paths will remain stable

self.chroot_cwd and do it from an appropriate `init_` hook

Reply:
Okay will try that